apps.toml

```diff
@@ -2306,6 +2306,12 @@ state = "working"
 subtags = [ "website" ]
 url = "https://github.com/YunoHost-Apps/my_webapp_ynh"
 
+[my_fastapi_app]
+category = "publishing"
+state = "working"
+subtags = [ "website", "api" ]
+url = "https://github.com/leonarf/FastAPI_ynh"
```

Comment:
Beware if your app id is `my_fastapi_app`, your repo should be named `my_fastapi_app_ynh`